libdevcore/CommonData.h

```diff
@@ -155,6 +155,14 @@ inline std::string formatNumber(bigint const& _value)
 		return _value.str();
 }
 
+inline std::string formatNumber(u256 const& _value)
```

Comment:
Would it make sense making this a template and/or supporting negative values as above? (In Julia it should not happen.)

Reply:
I tried this as a template, but the compiler will complain if you use `if (x < 0) return "-" + toHex(-x);` because neither `x < 0` nor `-x` makes any sense for an unsigned type. There probably is some template magic to avoid that problem, but I guess this is a rather easy solution...

Reply:
Right, `u256` should be unsigned.